hdfs-scheduler/src/main/java/org/apache/mesos/hdfs/config/ConfigServer.java

```diff
@@ -22,11 +22,7 @@
 import java.nio.charset.Charset;
 import java.nio.file.Files;
 import java.nio.file.Paths;
-import java.util.ArrayList;
-import java.util.HashMap;
-import java.util.Iterator;
-import java.util.List;
-import java.util.Map;
+import java.util.*;
 
```

Comment:
We prefer to import the specific classes instead of using *. Let's be consistent with the style.